feature-libs/product/multi-d/components/order-grid/order-grid.component.ts

```diff
@@ -1,10 +1,63 @@
 import { Component, ChangeDetectionStrategy } from '@angular/core';
+import { Product } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import {
+  GridVariantOption,
+  VariantsMultiDimensionalService,
+} from '../../core/services/variants-multi-dimensional.service';
```

Comment:
We can't have relative imports. This ones should be imported from `@spartacus/product/multi-d/core`

Reply:
I will fix all imports in the end of epic work